doc/api/single-executable-applications.md

```diff
@@ -4,7 +4,7 @@
 
 > Stability: 1 - Experimental: This feature is being designed and will change.
 
-<!-- source_link=lib/internal/main/single_executable_application.js -->
+<!-- source_link=lib/internal/main/embedding.js -->
```

Comment:
Good catch, thanks! Although, I think this is where the bulk of the SEA source code is now:
```suggestion
<!-- source_link=src/node_sea.cc -->
```